Source/storm/storm.cpp

```diff
@@ -78,7 +77,7 @@ bool SFileOpenFile(const char *filename, HANDLE *phFile)
 		std::string path = *SBasePath + filename;
 		for (std::size_t i = SBasePath->size(); i < path.size(); ++i)
 			path[i] = AsciiToLowerTable_Path[static_cast<unsigned char>(path[i])];
-		result = SFileOpenFileEx((HANDLE)nullptr, path.c_str(), SFILE_OPEN_LOCAL_FILE, phFile);
+		result = SFileOpenFileEx((HANDLE) nullptr, path.c_str(), SFILE_OPEN_LOCAL_FILE, phFile);

```

Comment:
Seems like HANDLE is a typedef void*. Don't need a cast.
But you could do a 
constexpr HANDLE INVALID_HANDLE = nullptr;
or similar somewhere.

Reply:
Ouside scope